src/Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier/Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier.DesignTime.csproj

```diff
@@ -18,44 +18,33 @@
     <DocumentationFile>$(DropRootDir)\ReferenceAssemblies\$(AssemblyName).xml</DocumentationFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <DebugSymbols>false</DebugSymbols>
-    <DebugType>pdbonly</DebugType>
```

Comment:
Does it matter if UnixPortSupplier is not always `pdbonly`?

Reply:
There is no real code in this module, so I believe that is why this is done.